src/CMakeLists.txt

```diff
@@ -743,7 +745,7 @@ if (WITH_MGR)
   add_executable(ceph-mgr ${mgr_srcs}
                  $<TARGET_OBJECTS:heap_profiler_objs>)
   target_include_directories(ceph-mgr SYSTEM PRIVATE "${PYTHON_INCLUDE_DIRS}")
-  target_link_libraries(ceph-mgr osdc client global-static ceph-common
+  target_link_libraries(ceph-mgr osdc client global-static common
```

Comment:
given `target_link_libraries(global-static common)` in global/CMakeLists.txt, is it necessary to list common for these executables?

Reply:
technically no, but we cannot rely on this fact that `global-static` depends on `common`.  strictly speaking, it's the internals of `global-static`.